example.env

```diff
@@ -0,0 +1,4 @@
+# copy and paste this to `.env`, then set values as appropriate
+ADMIN_USER="define your username"
+ADMIN_PASSWORD="define your password"
+mongoHost="mongo"
```

Comment:
Sorry to be a pain, but can you please change `mongoHost` to use uppercase as well so that all config values follow the same naming convention